### ISSUE TITLE
picker TypeError: Cannot read property 'value' of undefined

### ISSUE THREAD

**Issue by noumanDev** [GeekyAnts/NativeBase#1918]
## react-native, react and native-base version
react-native: 0.54.4
native-base: 2.4.1

 ## Expected behaviour
Should work

## Actual behaviour
Error : Cannot read property 'value' of undefined

## Steps to reproduce (code snippet or screenshot)
```
<Picker
      iosHeader="Select one"
      mode="dropdown"
      selectedValue={this.state.selected1}
      onValueChange={this.onValueChange.bind(this)}
    >
      <Picker.Item label="Wallet" value="key0" />
    </Picker>
```
## Screenshot of emulator/device
![error screen](http://icecream.me/uploads/669d63fc84a027df822e71e93bda9c60.png)

## Is the bug present in both ios and android or in any one of them?
i am working only on ios, i dont know about android

## Any other additional info which would help us debug the issue quicker.
for multiple picker.items it works

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.5.0](https://github.com/GeekyAnts/NativeBase/tree/v2.5.0)

**Comment by sojinantony01** [comment]
"react-native": "0.60.4"

issue still exists
